src/Devices/Ethernet_TC.h

```diff
@@ -5,7 +5,12 @@
 
 #pragma once
 
-#include "Ethernet.h"
+#include <Arduino.h>
+#ifdef MOCK_PINS_COUNT
+#include <Ethernet_CI.h>
```

Comment:
The file <Ethernet_CI.h> begins with the following:

```
#include <Ethernet.h>
#ifdef MOCK_PINS_COUNT
```

This means that you can include it unconditionally and it will include <Ethernet.h> for you.